rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,183 @@
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src when there is a disagreements between core developers.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * php-src - Source code in https://github.com/php/php-src/.
+  * contributor - Anyone who has already contributed to php-src.
+  * core developer - Anyone with write commit access to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2).
+  * maintainer - person responsible for core subsystem (e.g. extension or SAPI) that is listed in the php-src EXTENSION file or the php-src OWNERS files if added in the future.
+  * issue - php-src GitHub issue, including pull requests.
+  * process changes - Any changes impacting the way how PHP project is managed.
+  * pull request - php-src GitHub pull request.
+  * RM - PHP release managers.
+  * user facing changes - Changes in PHP language and its semantics, user visible functions, classes, interfaces, constants and other user visible constructs.
+  * technical change - Any change to php-src that has no user-facing change, including the implementation details of an RFC-accepted user facing change.
+  * technical aspect - The portions of  pull request or commit in php-src or commit to php-src master and dev branches that would be technical changes.
+  * technical decision - Decision on a technical aspect.
+  * TC - PHP Technical Committee.
+
+To further clarify some of the defined terms, following subsection provides some useful examples.
+
+=== Changes ===
+
+It is important to clarify difference between user facing change and technical changes. As it is defined, the technical
+change is any php-src change that is not a user facing change. However, the implementation of user facing change is
+still a technical change. The meaning of that is that the part of the user facing change that defines the user
+interface is not a technical change.
+
+For example if there is a proposal to add accessors for PHP classes, then everything that is visible to user space
+like the constructs and their usage in PHP code is a user facing change. The actual C implementation and tests are
+a technical change.
+
+==== TC membership ====
+
+The TC consists of five elected members.  Any core developer is eligible to run for the TC.  One member of the TC,
+selected by the TC members themselves, is the TC Secretary.
+
+The TC Secretary is responsible for:
+
+  * Administering the election of the next set of TC members.
+  * Documenting and communicating any decisions of the TC.
+  * In case of a tie, the Secretary's vote will be considered the winning position.
```

Comment:
Not totally sure how I feel about the secretary being the one deciding the outcome of a tie. But hopefully this shouldn't happen often.

Reply:
Yeah I wouldn't expect this happen often as it would require either reduced number of TC members or some members abstaining. The main point is that it should always be decided by the TC so there should be someone in the TC to make that decision if there's a tie. I really don't want to add some exception that would allow fallback to RFC for the technical decision as this is exactly what we are trying to prevent.

Reply:
Agreed